onnxruntime/contrib_ops/cpu/transformers/logits_processor.h

```diff
@@ -141,10 +150,117 @@ class PresencePenaltyLogitsProcessor : public ILogitsProcessor<T> {
 template <typename T>
 class TimestampLogitsProcessor : public ILogitsProcessor<T> {
  public:
-  TimestampLogitsProcessor(int eos_token_id, int max_initial_timestamp_index);
+  TimestampLogitsProcessor(int eos_token_id, int max_initial_timestamp_index)
+      : eos_token_id_(eos_token_id), max_initial_timestamp_index_(max_initial_timestamp_index) {}
 
   void Process(const ISequences* sequences,
-               NextTokenScores<T>& next_token_scores) override;
+               NextTokenScores<T>& next_token_scores) override {
+    const int beg_token_id_ = eos_token_id_ + 107;
```

Comment:
Is there plan to move timestamp to CUDA as well?

Reply:
next step